nc/models.py

```diff
@@ -203,7 +203,7 @@ class StopSummary(pg.ReadOnlyMaterializedView):
 
     id = models.PositiveIntegerField(primary_key=True)
     year = models.IntegerField()
```

Comment:
I might remove `year` to 1) reduce the extra records created by the `date` and 2) I can just format the date to year like it's done for month and week. @copelco Any thoughts?

Reply:
I opened #189 as a suggested improvement for this